### DISCUSSION TITLE
How to use Context api (or any state management library ) inside React Query ??🤔

### DISCUSSION THREAD

**Discussion by leamandeep** [TanStack/query#3651] [Q&A]
Hey I am facing bit difficulties in using react query with context api ?? I can't use both at same time . 
Is there any possible solutions ?? Please help me 🙏

**Comment by TkDodo** [comment]
I honestly don't understand what that means. Please try to tell us what you are trying to achieve, and at best provide a codesandbox reproduction of your problem.

**Reply by leamandeep** [reply]
ok I will provide you codesandbox reproduction of my problem.

**Reply by leamandeep** [reply]
@TkDodo Thanks 🙏 for giving me your precious time . But problem solved . Since i am unable to use codesandbox properly . So , I am leaving code file below 👇
[react-query.zip](https://github.com/tannerlinsley/react-query/files/8793542/react-query.zip)
.
